features/buyer/direct-award.feature

```diff
@@ -1,7 +1,103 @@
 @direct-award
 Feature: Direct Award flows
+  
+Scenario: Unauthenticated user can save a search after logging in
+  Given I have a production buyer user
+  And I am on the /g-cloud/search page
```

Comment:
All of these scenarios have the user saving an empty search to start with.  Might be nice to have some where they enter keywords, click some filters and then save a new search.